components/atoms/card.tsx

```diff
@@ -24,79 +24,52 @@ interface CountyBreakdownCardProps {
   };
   onPress?: () => void;
 }
-export const Card: FC<CountyBreakdownCardProps> = ({
+
+export const Card: FC<CardProps> = ({
   type,
-  padding: {h = 16, v = 16, r = 16} = {},
+  padding: {h = 16, v = 16, r} = {},
```

Comment:
Some components have explicitly set the right margin, for this reason, the right padding prop was introduced.

The best thing we can do here, I think, it's to remove the `padding` prop, set the padding default values using `style` and add a `style` prop to Card that allows the user to override any CSS property, including the padding values.

Reply:
I would have some default values just to avoid writing them all the time. I guess that if we need left margin changed we would just add it as a prop.
Usually with this sort of components I'd avoid passing the style in just to enforce the purpose of it.